about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/f25b74c76b.js" crossorigin="anonymous"></script>
+    <title>capstone-project</title>
+</head>
+<body>
+  <header id="dsktop-header">
+    <nav id="nav1" title="first-navigation">
+      <ul>
+        <li><a class="fa-brands fa-facebook-f"></a></li>
+        <li><a class="fa-brands fa-twitter"></a></li>
+        <li><a href="#" class="nav1item">English</a></li>
+        <li><a href="#" class="nav1item">My Page</a></li>
+        <li><a href="#" class="nav1item">Logout</a></li>
+      </ul>
+    </nav>
+    
+    <nav id="nav2" title="second-navigation"> 
+      <a class="main-logo" href="#about"><img src="images/icecream.png" alt=""></a> 
+      <a class="nav2Item" href="./index.html">Home</a>
+      <a class="nav2Item" href="./index.html">Packages</a>
+      <a class="nav2Item" href="#join">Join</a>
+      <a class="nav2Item" href="#sponsor">Sponsor</a>
+      <a class="nav2Item" href="#news">Recipes</a>
+      <a class="mainItem" href="./index.html">workshops</a> 
+    </nav>   
+</header>
+
+<header id="mobile-header">
+  <nav class="mobile-nav"> 
+      <ul onclick="menuPopup()" id="navList" title="navigation-list">
+        <li><a class="navItem" href="./index.html">Home</a></li>
+        <li><a class="navItem" href="./index.html">Our Packages</a></li>
+        <li><a class="navItem" href="#join">Join</a></li>
+        <li><a class="navItem" href="#sponsor">Sponsor</a></li>
+        <li><a class="navItem" href="#news">Recipes</a></li>
+        <li><a class="navItem navItem-main" href="#about">workshops</a></li>
+      </ul> 
+  </nav>   
+  <nav>
+    <div onclick="menuPopup()" id="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div> 
+    <div onclick="menuPopup()" id="closeButton"> &times; </div>
```

Comment:
- [ ] Just as I suggested above, It will be of good practice to move this functionality in the `main.js` file :+1: :smile: